operator/cilium_node.go

```diff
@@ -32,15 +32,15 @@ import (
 
 var k8sCiliumNodesCacheSynced = make(chan struct{})
 
-func startSynchronizingCiliumNodes(nodeManager allocator.NodeEventHandler) {
+func startSynchronizingCiliumNodes(nodeManager allocator.NodeEventHandler) error {
```

Comment:
can you pass `apiextensionsK8sClient` as an argument of this function and add documentation stating the function will wait until the creation of the CRD for `CiliumNodes`?

Reply:
done